src/components/Order.tsx

```diff
@@ -71,15 +73,9 @@ export const Order = ({ trainInfo, userData }: Props) => {
                   <Typography>
                     <strong>Personal Information</strong>{' '}
                   </Typography>
-                  <Typography>
-                    {userData.passengerInformation.name}{' '}
-                    {userData.passengerInformation.surname}
-                  </Typography>
-                  <Typography>
-                    1*Youth(16-26 yrs) ({userData.passengerInformation.discount}
-                    )
-                  </Typography>
-                  <Typography>{userData.passengerInformation.email}</Typography>
+                  <Typography>Name Surname</Typography>
```

Comment:
?? What we need to put there??

Reply:
Just remove changes that you did in this file. All changes that you did here are not correct we should use information  from userData object as it was before
You can see what are the difference if you'll go to the order overview page, with your changes there will be Name Surname at user info tab instead of correct name from user.json file (Jakub Liptak or smt like this)